ui/src/app/pages/signed-in/access-renewal-notification.spec.tsx

```diff
@@ -27,6 +29,8 @@ const allModules = [
   AccessModule.PUBLICATIONCONFIRMATION,
 ];
 
+const EMPTY_STRING = '';
```

Comment:
Could rename this to NO_BANNER_NO_COLOR

Reply:
My preference would be two variables, one for banner and one for color.

I'm fine with whatever you prefer.

Reply:
my pref here would be to leave these values undefined, since they are not checked